src/YoloDev.Expecto.TestSdk/adapter.fs

```diff
@@ -102,3 +102,34 @@ type VsTestAdapter() =
 
       Execution.runTests logger runSettings frameworkHandle sources
       |> Async.RunSynchronously
+
+/// Defines the identity of the Expecto extension for Microsoft.Testing.Platform.
+type ExpectoExtension() =
+    interface Microsoft.Testing.Platform.Extensions.IExtension with
+      member _.Uid = nameof(ExpectoExtension)
+      member _.Version = "0.14.4"
```

Comment:
This should be improved to not be hardcoded.

Reply:
The package does set versioning, and I checked that the binaries do have AssemblyInformationalVersion set during the build, so you could read the version from that attribute:

```fsharp
member _.Version = System.Reflection.Assembly.GetExecutingAssembly().GetCustomAttribute<System.Reflection.AssemblyInformationalVersionAttribute>().InformationalVersion
```